pages/login-page/login-page.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
You can add desc. in your meta tag like author name, page details, page name, etc

Reply:
Oh, new thing to me. Will learn and use in future.